Index: index.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\necho \"test\";\r\n\r\n//testing the version control\r\n\r\necho \"lets try that again\";\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- index.php	(revision 4dcdbe83680e547bcf88674184527b17de9e5d90)
+++ index.php	(date 1614769436840)
@@ -4,4 +4,3 @@
 //testing the version control
 
 echo "lets try that again";
-
